assets/js/components/Dashboard/PaMessagesPage.tsx

```diff
@@ -161,19 +161,19 @@ const PaMessagesPage: ComponentType = () => {
             </section>
             <FilterGroup
               className="mb-5"
-              header="Filter by message state"
+              header="Message state"
```

Comment:
I think this change is a step backwards in terms of accessibility (posted a comment in the Miro design as well). There aren't any section headers or other landmarks here, so without the words "filter by", there's no way for a screen reader to know that these controls are filters. (As a side note, the controls themselves aren't really marked up properly either, which we should think about fixing as a separate effort)

Reply:
That makes perfect sense. I'll make those changes from what we discussed in standup and update the ticket to make it consistent on the other page.

Reply:
Switched the language back to be "Filter by" but also added it on the static template page to be consistent (with approval from design).